requirements3.txt

```diff
@@ -0,0 +1,88 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile --output-file requirements3.txt requirements.in requirements-dev.in
```

Comment:
I know this is an auto-generated comment, but the inputs listed are the same as for `requirements2.txt`, so it would be unclear to someone reading this how they might update this file for both python 2 and 3. It's probably worth mentioning how to accomplish this in the docs somewhere.

Reply:
After speaking with @jbeezley he would like to transition girder worker to use [pipfile](https://github.com/pypa/pipfile) so i'm not sure how long this will be around - see also https://github.com/girder/girder_worker/issues/265

Reply:
Ok, we'll punt on this.